src/commands/publish.ts

```diff
@@ -520,6 +489,25 @@ export async function publishMain(argv: PublishOptions): Promise<any> {
     }
   }
 
+  logger.info(`Looking for publish state file for ${newVersion}...`);
+  const publishStateFile = `.craft-publish-${newVersion}.json`;
+  const earlierStateExists = existsSync(publishStateFile);
+  let publishState: PublishState;
+  if (earlierStateExists) {
+    logger.info(`Found publish state file, resuming from there...`);
+    publishState = JSON.parse(readFileSync(publishStateFile).toString());
+    targetList = new Set(getAllTargetNames());
```

Comment:
We should probably rename `targetList` to `targetSet` or similar, to avoid any confusion later e.g. regarding ordering.